EventFilter/L1TXRawToDigi/python/caloLayer1Stage2Digis_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+l1tCaloLayer1Digis = cms.EDProducer(
+    'L1TCaloLayer1RawToDigi',
+    fedRawDataLabel = cms.InputTag("rawDataCollector"),
+    FEDIDs = cms.vint32(1354, 1356, 1358),
```

Comment:
in longer term, the fed-id lists usually come from the data formats headers for FEDRawData